cmd/minishift/cmd/util/oc.go

```diff
@@ -92,3 +92,20 @@ func SetOcContext(profileName string) error {
 
 	return nil
 }
+
+//RemoveCurrentContext removes the current context from `machinename_kubeconfig`
+func RemoveCurrentContext() error {
+	requestedOpenShiftVersion := viper.GetString(configCmd.OpenshiftVersion.Name)
+	ocPath := CacheOc(clusterup.DetermineOcVersion(requestedOpenShiftVersion))
```

Comment:
During my testing of switching between non existing profiles I did not see oc getting cached. Which is a good thing as we should not download oc while running `profile set command`. But as per code it should download the oc binary. Looking in to the code to understand what is happening here.

Reply:
How did you test it?